addons/userconfig/cba_settings.sqf

```diff
@@ -16,18 +16,31 @@ force ace_finger_enabled = true;
 
 force ace_gforces_enabledFor = 0;
 
-force ace_goggles_effects = 0;
-
 force ace_hearing_enableCombatDeafness = false;
 
 force ace_map_mapShowCursorCoordinates = true;
 
-force ace_medical_ai_enabledFor = 0;                      // Medical system not enabled for AI
-force ace_medical_feedback_painEffectType = 2;            // Pain effect type is "Chromatic Abberation".
-force ace_medical_gui_enableMedicalMenu = 1;              // Enable medical menu.
```

Comment:
Restore

Reply:
Displaying medical menu is a player-side preference, no need to force it

Reply:
Players forget to enable it, therefor it's nice to have it as an option out of the box